src/static/riot/search.tag

```diff
@@ -656,7 +659,12 @@
             padding-bottom 0 !important
 
         .content-desktop
-            margin-top 0
+            margin 10px auto !important
+            max-width 1750px
+
+        .ultrawide
```

Comment:
Should this be a more global class? Maybe on some other pages we'll want this as well, not just search?

Reply:
Added an ultrawide class to the base styling template.